kitchen/ph5toms

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
-#   Version: 2016.133
-EXEC='ph5tomsAPI'
+#   Version: 2017.167
+EXEC='ph5toms'
```

Comment:
dataselect probably wont work since we have other clients for outputting segy in receiver order and event order. Maybe a good name will come to me over the weekend